blackbox/docs/config/secure.rst

```diff
@@ -0,0 +1,82 @@
+.. highlight:: sh
+
+.. _conf-secure-settings:
+
+===============
+Secure Settings
+===============
+
+Some settings require additional protection. For example: passwords, access 
+keys, and so on. When handling this type of data, it is typically best practice 
+not to rely on file system permissions, and not to store it on disk using plain 
+text.
+
+For sensitive configuration data, we provide a keystore_ along with the 
+``crate-keystore`` tool to manage the data in the keystore_.
+
+.. NOTE::
+
+    All commands here should be run as the user which will run CrateDB.
+
+.. NOTE::
+
+    Changes to the keystore only take effect after restarting CrateDB.
+
+.. NOTE::
+
+    The CrateDB keystore currently provides only obfuscation, as password 
+    protection is not currently implemented.
```

Comment:
I am not sure what this means. can you expand or reword? what do we mean by obfuscation specifically?

Reply:
@nomi I'm not specifically sure what you're after here tbh. I attempted a reword, hopefully, that's better. 
(as stated before, "obfuscation" is an established term and defining it here is like providing a definition for "permission", "file system" or "setting" )
In case the rewording is not better please be more specific about what you'd like changed